### PR TITLE
Fixing test failure

### PR DESCRIPTION
**Pull request by Nishith**

### CODE REVIEW

**Comment by Nishith** [comment]
Pending build http://builds.cask.co/browse/CDAP-DUT4609-1

File: cdap-app-fabric/src/test/resources/cdap-security.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+<!--
+  Copyright © 2014 Cask Data, Inc.
```
Comment (chtyim) [inline cdap-app-fabric/src/test/resources/cdap-security.xml:4]: 2016

Comment (chtyim) [inline cdap-app-fabric/src/test/resources/cdap-security.xml:4]: You can set IntelliJ to use current date year

**Comment by bdmogal** [comment]
@Nishith let's close this PR. there should be a better way to fix these tests - should be able to bind a NoOpSecureStore for tests that don't need it.

**Comment by anwar6953** [comment]
I'd recommend an UnsupportedSecureStore instead, which throws exceptions in its method calls, if we feel that it won't get used from tests.

**Pull request closed without merging**